messaging-activemq/src/main/resources/org/wildfly/extension/messaging/activemq/LocalDescriptions.properties

```diff
@@ -381,6 +381,7 @@ server.journal-min-files=How many journal files to pre-create.
 server.journal-node-manager-store-table=Name of the JDBC table to store the node manager.
 server.journal-page-store-table=Name of the JDBC table to store pages.
 server.journal-pool-files=The number of journal files that can be reused. ActiveMQ will create as many files as needed however when reclaiming files it will shrink back to the value (-1 means no limit).
+server.journal-file-open-timeout=The timeout (in seconds) for opening journal files.
```

Comment:
What happens if the timeout is exceeded?

Reply:
Value <= 0 means fail immediately. I updated the description.

Reply:
Sorry, looks like I didn't read the question carefully. If journal file cannot be opened, the ActiveMQ server shuts down.